keyvi/CPPLINT.cfg

```diff
@@ -1,2 +1,3 @@
 linelength=120
 root=keyvi/include
+filter=-build/include_subdir
```

Comment:
can you explain, afaik that enforces always using full qualified directories. Not to bad.

Reply:
Agree! 
But as we include `#include "utf8.h"` from CMake include paths, and there it's not hierarchy, I couldn't do `#include "utf8/utf8.h"`

The alternative could be to change `CMakeLists.txt` to have `include_directories(keyvi/3rdparty)` 

Or to use includes from `utf8.h` directly. 
```
#include "utf8/checked.h"
#include "utf8/unchecked.h"
```
Didn't have strong opinion which is the best, so decided to send a PR and we discuss.

Reply:
FYI: the same change was needed in https://github.com/KeyviDev/keyvi/pull/44 as well.

Reply:
ah I see, so it is because the 3rdparty dependency isn't integrated like tpie, rapidjson and the others.

this is from utf8cpp, origin http://utfcpp.sourceforge.net/ (currently offline)

there are several github versions, e.g. https://github.com/ledger/utfcpp - does not look official.

So the alternative would be to fix the 3rdparty paths, introducing subdirectories there, but the same problem applies to md5 and tiny-process.

I suggest: keep this fix for later.